tests/components/rest/test_switch.py

```diff
@@ -446,6 +446,9 @@ async def test_entity_config(hass: HomeAssistant) -> None:
     entity_registry = er.async_get(hass)
     assert entity_registry.async_get("switch.rest_switch").unique_id == "very_unique"
 
+    async_fire_time_changed(hass, utcnow() + SCAN_INTERVAL * 2)
+    await hass.async_block_till_done()
+
```

Comment:
Why is this needed?

Reply:
Only name template is evaluated during startup so the other attributes needs a refresh to be set

Reply:
That seems odd. Why are the templates not rendered once the entity is added?